tests/steps/test_security_indicators.py

```diff
@@ -139,3 +153,22 @@ def test_security_indicator_scoring_missing_stable(self, recommendation_type) ->
             result[1][0]["message"] == "flask===0.12.0 on https://pypi.org/simple has no "
             "gathered information regarding security."
         )
+
+    @pytest.mark.parametrize("recommendation_type", [RecommendationType.SECURITY])
+    def test_security_indicator_with_high_high(self, recommendation_type) -> None:
```

Comment:
Missing type annotations, but I think there is no need to have this parametrized and can be directly assigned in the test function (just one parameter).